api/eventTypes.js

```diff
@@ -0,0 +1,14 @@
+let array = [ 
+    {id : "CoP"},
+    {id : "landl"},
+    {id : "learningEvents"},
+    {id : "other"},
```

Comment:
Shouldn't it also contain `labels: ['blabla']`? Or is yet to be added?

Reply:
We are going to add in alias once we agree on the different variations that can come in :)